docs/faq/general/why-cant-i-see-my-organization.md

```diff
@@ -16,4 +16,8 @@ If you don't see your organization when [adding your organization on Codacy](../
 
 1.  Revoke Codacy's OAuth application on the Git provider website and log in again to Codacy.
 
+1.  **If you're using GitHub,** check if you've reached the [maximum number of organization event webhooks](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks){: target="_blank"} and delete any webhooks that are no longer needed.
```

Comment:
Here I would prefer not to mention the current limit of 20 webhooks explicitly, since the number may change in the future.